docs/getting-started/release-notes/v2_13_0.md

```diff
@@ -50,10 +67,12 @@ Zowe Version 2.13.0 contains the enhancements that are described in the followin
 Zowe Version 2.13.0 contains the bug fixes that are described in the following topics.
 
 ### Zowe installation and packaging
-
+- Workflow files in the `Zowe PAX` are now `ASCII-encoded`. ([#3591](https://github.com/zowe/zowe-install-packaging/issues/3591))
```

Comment:
It would be good to explain how, specifically, this is a bug fix. What bug is it resolving?

Reply:
Thanks Ana for a quick review. Made changes to this bullet and committed the changes. Please review and let me know.